add the one-line script to install AM collector (#19)
aspenmesh/aspenmesh-charts: docs/aspenmesh-collector.MD

## Installation

### One-line script installation

> The Aspen OpenTelemetry Collector can be installed by a one-line script.

Reviewer: I vaguely remember Joanna saying something about us removing "OpenTelemetry" from our collector name? let's check, and do that here if needed

```bash

> /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/aspenmesh/aspenmesh-charts/main/scripts/aspenmesh-collector-setup.sh)" -- --endpoint <remote-write-endpoint> --apiKey <user-group-api-key>

Author: Unfortunately, `https://git.io/` is no longer accepting new links for shortening URL. Are we OK with the github raw content URL? Is it worth to search another free and legit third-party website?

Author: I think `/bin/bash` works for most users? Windows users need to either install `sh` or `bash shell` anyway

```

> You can also follow the details for each installation step.

Reviewer: suggested wording:

```suggestion
Alternatively, you can install using the following steps
```
